.dockerignore

```diff
@@ -4,5 +4,5 @@
 htmlcov
 dist
 **/__pycache__
-**/*.pyc
-cpp/build
\ No newline at end of file
+cpp/build
+Dockerfile
```

Comment:
Added this so that when I change Dockerfile, the complete MAGE folder is not copied again - the whole process is lot slower for local dev.

Reply:
https://docs.docker.com/develop/develop-images/build_enhancements/

This is how built might be enhanced in terms of speed

Reply:
I tested it out, maybe 40 seconds faster. I will push it to GitHub, maybe it helps there more.